Unix/codec/mof/parser/mof.qualifiers.h

```diff
@@ -7,334 +7,335 @@
+L"    Scope(property);\n";
+
+//
+// CIM-2.26.0: qualifiers_optional.mof
+//
+const wchar_t cOptqualifiers[]=
+L"// Copyright (c) 2007 DMTF.  All rights reserved.\n"
+L"// <change cr=\"ArchCR00071.005\" type=\"add\">Add new \n"
+L"//DisplayDescription qualifier.</change>\n"
+L"// ==================================================================\n"
+L"// Optional Qualifiers\n"
+L"// Defined in the CIM Infrastructure Specification Section 2.5.3 \n"
+L"// ==================================================================\n"
+L"\n"
+L"Qualifier Alias : string = null, \n"
+L"    Scope(property, reference, method), \n"
+L"    Flavor(EnableOverride, ToSubclass, Translatable);\n"
+L"\n"
+L"Qualifier Delete : boolean = false, \n"
+L"    Scope(association, reference);\n"
+L"\n"
+L"Qualifier DisplayDescription : string = null, \n"
+L"    Scope(any), \n"
+L"    Flavor(EnableOverride, ToSubclass, Translatable);\n"
+L"\n"
+L"Qualifier Expensive : boolean = false, \n"
+L"    Scope(any);\n"
+L"\n"
+L"Qualifier Ifdeleted : boolean = false, \n"
+L"    Scope(association, reference);\n"
+L"\n"
+L"Qualifier Invisible : boolean = false, \n"
+L"    Scope(class, association, property, reference, method);\n"
+L"\n"
+L"Qualifier Large : boolean = false, \n"
+L"    Scope(class, property);\n"
+L"\n"
+L"Qualifier PropertyUsage : string = \"CurrentContext\", \n"
+L"    Scope(property);\n"
+L"\n"
+L"Qualifier Provider : string = null, \n"
+L"    Scope(any);\n"
+L"\n"
+L"Qualifier SyntaxType : string = null, \n"
+L"    Scope(property, reference, method, parameter);\n"
+L"\n"
+L"Qualifier Syntax : string = null, \n"
+L"    Scope(property, reference, method, parameter);\n"
+L"\n"
+L"Qualifier TriggerType : string = null, \n"
+L"    Scope(class, association, indication, property, reference, method);\n"
+L"\n"
+L"Qualifier UnknownValues : string[], \n"
+L"    Scope(property), \n"
+L"    Flavor(DisableOverride, ToSubclass);\n"
+L"\n"
+L"Qualifier UnsupportedValues : string[], \n"
+L"    Scope(property), \n"
+L"    Flavor(DisableOverride, ToSubclass);\n"
+L"\n";
+
+//
+// MSFT_Qualifiers.mof
+//
+const wchar_t cMsftqualifiers[]=
+L"//============================================================================\n"
+L"// Copyright (C) Microsoft Corporation, All rights reserved. \n"
+L"//============================================================================\n"
+L"//\n"
+L"\n"
+L"Qualifier Stream : boolean = false, \n"
+L"    Scope( method, parameter), \n"
+L"    Flavor(DisableOverride, ToSubclass);\n"
+L"\n"
+L"Qualifier ClassVersion : string = null, \n"
+L"    Scope(class, association, indication), \n"
+L"    Flavor(EnableOverride, Restricted);\n";
+
+//
+// wmiqualifiers.mof
+//
+#endif
+
+/*=============================================================================
+**
+** Declare global qualifer declarations
+**
+=============================================================================*/
+extern MI_QualifierDecl gQualifiers[];
+
+#endif /* _mof_qualifiers_h */
+
```

Comment:
Both old and new have a large block enclosed by #if 0...#endif.   What's different?